tests/test_opds_import.py

```diff
@@ -131,24 +131,25 @@ def test_extract_next_links(self):
 
     def test_extract_last_update_dates(self):
         importer = OPDSImporter(self._db, DataSource.NYT)
+
```

Comment:
A bunch of these tests were changed because they do something to every <entry> tag and messages are no longer <entry> tags.